GW2EIEvtcParser/EIData/ProfHelpers/Elementalist/ElementalistHelper.cs

```diff
@@ -239,6 +241,12 @@ internal static class ElementalistHelper
                 .WithBuilds(GW2Builds.February2024NewWeapons),
             new Buff("Shattering Stone", ShatteringStoneBuff, Source.Elementalist, BuffStackType.StackingConditionalLoss, 25, BuffClassification.Other, BuffImages.ShatteringStone)
                 .WithBuilds(GW2Builds.February2024NewWeapons),
+            // Spear
+            new Buff("Seethe", SeetheBuff, Source.Elementalist, BuffClassification.Offensive, BuffImages.MonsterSkill),
```

Comment:
Are those buffs you can share with others? Otherwise the classification should be Other.

Reply:
i am not sure i haven't tested everything with allies, those that increased damage got added to offensive

Reply:
Ripple: The cooldown of your next spear skill is reduced.
Energize_: Your next spear skill always critically strikes.

I'm quite sceptical you give those to the allies, they look like personal buffs